cunumeric/array.py

```diff
@@ -1959,9 +1959,12 @@ def compress(self, condition, axis=None, out=None):
                 category=RuntimeWarning,
             )
             condition = condition.astype(bool)
+
         if axis is None:
             axis = 0
             a = self.ravel()
+        else:
+            axis = normalize_axis_index(axis, self.ndim)
```

Comment:
According to the numpy docs, negative axes are acceptable:

https://numpy.org/doc/stable/reference/generated/numpy.compress.html